src/da2013.rs

```diff
@@ -48,19 +50,27 @@ impl Da2013 {
 
         // HID report number
         buf.push(0);
+
         // Status
         buf.push(0);
+
         // Padding
         let mut zeros = vec![0; 3];
         buf.append(&mut zeros);
+
+        // Data
```

Comment:
I think adding this comment here is a bit too verbose, the ones I already have are mostly for the stuff that is not obvious just by looking at the code itself.